apps/omg_eth/lib/omg_eth/release_tasks/set_contract.ex

```diff
@@ -85,20 +86,25 @@ defmodule OMG.Eth.ReleaseTasks.SetContract do
 
     txhash_contract = get_env(network <> "_TXHASH_CONTRACT")
     authority_address = get_env(network <> "_AUTHORITY_ADDRESS")
-    contract_address = get_env(network <> "_CONTRACT_ADDRESS")
+    env_contract_addresses = get_env(network <> "_CONTRACT_ADDRESS")
+    contract_addresses = if env_contract_addresses, do: Jason.decode!(env_contract_addresses), else: nil
```

Comment:
with the problems that devops has deploying this. pls let us move away from encoding JSON into the environment variables. please lets find another format to pass in the value. maybe comma separated values? 
or descriptive environment variables for each contract, like we had before? lets not break the pattern.

Reply:
For future ref: sth similar proposed in `temp/948_fix_JSON_sys_env` branch, went with the latter option.